packages/coinstac-ui/app/render/utils/boot/configure-logger.js

```diff
@@ -11,20 +12,29 @@ const mainLogger = electron.remote.require('app/main/utils/expose-app.js')().log
  * defaults)
  * @returns {Function} logger
  */
-function getLogger(consoleMethod, winstonMethod) {
+function getLogger(consoleMethod, winstonMethod, logger) {
   return (...args) => {
     console[consoleMethod].apply(console, args); // eslint-disable-line no-console
-    mainLogger[winstonMethod].apply(null, ['ui', ...args]);
+    logger[winstonMethod].apply(null, ['ui', ...args]);
   };
 }
 
 module.exports = function configureLogger() {
-  app.logger = {
```

Comment:
Can’t remember why we’re doing this. Seems pretty convoluted: a separate logger (wrapper?) for every method? This essentially decorates Winston methods. Wonder if there's another way around it.